go_minor.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+#set the source path of gvm. $HOME points to '/root'
+[[ -s "$HOME/.gvm/scripts/gvm" ]] && source $HOME/.gvm/scripts/gvm
+export  CGO_ENABLED=0
+
+declare -a versions=( '1.1' '1.1.2' '1.2' '1.2.2' '1.3' '1.3.3' '1.4' '1.4.3' )
+for version in "${versions[@]}"
+  do
+     # Install Go $version
+     echo "================= Install Go $version==================="
+      gvm install go$version && gvm use go$version && go install -a std #&& go get -u github.com/tools/godep
```

Comment:
Please fix the indent (extra space at start of line), and remove the commented part of the command.